NetworkOfPrivateClinics/NetworkOfPrivateClinics/DailyRoutine.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NetworkOfPrivateClinics
+{
+    public class DailyRoutine
+    {
+        public Dictionary<TimeOnly, Patient> ListOfDailyAppointments { get; set; }
+
+        public DailyRoutine(string startWorkingDay, string endWorkingDay)
+        {
+            StartWorkingDay = ConvertToTimeOnly(startWorkingDay);
+            EndWorkingDay = ConvertToTimeOnly(endWorkingDay);
+            ListOfDailyAppointments = GenerateDailyRoutine(StartWorkingDay, EndWorkingDay);
+        }
+
+        public readonly TimeOnly StartWorkingDay;
+        public readonly TimeOnly EndWorkingDay;
+
+        private TimeOnly ConvertToTimeOnly(string time) => TimeOnly.Parse(time, new CultureInfo("uk-UA"));
+
+        private Dictionary<TimeOnly, Patient> GenerateDailyRoutine(TimeOnly startWorkingDay, TimeOnly endWorkingDay)
+        {
+            TimeOnly currentTime = startWorkingDay;
+            Dictionary<TimeOnly, Patient> generatedList = new();
+            while (currentTime < endWorkingDay)
+            {
+                generatedList.Add(currentTime, null);
+                currentTime = currentTime.AddHours(1);
+            }
+            return generatedList;
+        }
+
+        public Patient this[TimeOnly time]
+        {
+            get => ListOfDailyAppointments[time];
+            set => ListOfDailyAppointments[time] = value;
+        }
+    }
```

Comment:
I'm afraid it won't be convenient to use this class. Imagine the doctor has appointment on 8:00 and 8:15.
Your dictionary will contain keys 8 and 8:15. I want to make appointment at 8:10 and it will probably take 15 minutes. How do we handle these clashes?